geograph/geotimeline.py

```diff
@@ -9,8 +9,8 @@
 import pandas as pd
 import xarray as xr
 
-from src.binary_graph_operations import NodeMap, identify_graphs
-from src.geograph import GeoGraph
+from geograph import GeoGraph
```

Comment:
See comment above (module vs. package), unless you have elevated GeoGraph to be directly importable?

Reply:
Yes, I have - check `__init__.py` in the top level. I felt it was better to be able to do `from geograph import GeoGraph` or `import geograph` then `geograph.GeoGraph` rather than adding the extra `geograph` in the middle.

Reply:
Nice, yep that's definitely cleaner!